django_usaid_clinicfinder/clinicfinder/tasks.py

```diff
@@ -299,6 +287,48 @@ def run(self, lookuppointofinterest_id, **kwargs):
                  via Celery.',
                 exc_info=True)
 
+    def format_match_aat(self, match):
+        return "%s (%s)" % (
+            match.get('OrganisationName'),
+            match.get('FullAddress'))
+
+    def search_aat(self, lookuppoi):
+        url = (
+            "%(url)s?username=%(username)s&password=%(password)s&meters=50000"
+            "&category=%(category)s&x=%(x)s&y=%(y)s") % {
+                'url': settings.AAT_API_URL,
+                'username': settings.AAT_USERNAME,
+                'password': settings.AAT_PASSWORD,
+                'category': settings.AAT_CATEGORIES,
```

Comment:
You can't just pass in `AAT_CATEGORIES` here -- it's a dictionary and you need the category number. You will need to look up the category name in the dictionary and get its category number and then pass that in instead.

E.g.:
```python
def get_aat_category_id(self, search):
    category = 'mmc'
    for cat_name, cat_value in settings.AAT_CATEGORIES.items():
        if search.get(cat_name, False):
            category = cat_name
            break
    return settings.AAT_CATEGORIES[category]
```